src/androidTest/java/com/owncloud/android/AbstractIT.java

```diff
@@ -97,9 +97,9 @@ public static void beforeAll() throws InterruptedException,
 
         OwnCloudClientManagerFactory.setUserAgent("Mozilla/5.0 (Android) Nextcloud-android/1.0.0");
 
-        nextcloudClient = new NextcloudClient(url, context);
-        nextcloudClient.credentials = Credentials.basic(loginName, password);
-        nextcloudClient.userId = loginName; // for test same as userId
+        String userId = loginName; // for test same as userId
+        String credentials = Credentials.basic(loginName, password);
+        nextcloudClient = new NextcloudClient(url, userId, credentials, context);
```

Comment:
couldn't you just use `loginName`, creating a second variable with the same value and then using both here seems a bit confusing.

Reply:
True, but as this is for test only, I wanted to have it clear that userId and login credentials are normally not the same.